### ISSUE TITLE
Nether biomes in overworld when switching from 1.15 to 1.16

### ISSUE THREAD

**Issue by dranorter** [TerraformersMC/Traverse#38]
Steps to reproduce: Create a world in 1.15.2, using Traverse 2.1.14 build 37. Save, then open it in 1.16.1, with Traverse 2.2.11 build 23. Explore the overworld and you should find a region fairly quickly where nether biome particles will appear in the air, nether mobs will spawn, and the fog color will correspond to a nether biome. Pressing F3 shows the current biome is a nether biome.

Seemingly relevant line from the log:

[03:20:01] [main/WARN]: minecraft:nether missing from registry, but requested!

I don't know if this is properly as issue or not, but I'd appreciate any insight on what's going on since I'm encountering this in a world that I'd like to bring to 1.16.1.

https://paste.ee/p/Plrfk

![2020-08-29_03 22 42](https://user-images.githubusercontent.com/14154658/91631376-189d6580-e9a7-11ea-864b-0c1cccbdb804.png)

**Comment by dranorter** [comment]
Seems like this can be "fixed" by loading and saving the 1.15 world without Traverse loaded. Then, load it up in 1.16 with Traverse loaded, and biomes actually line up with where they were inside 1.15 (from what I've seen so far). I may end up compiling a version of the 1.15 Terrestria which has all the blocks but none of the biomes, in order to move a world over without losing blocks.

**Comment by ghost** [comment]
Yeah we've been getting a lot of issues similar to this one. We believe that it's an issue with minecraft itself, but we aren't sure. We're still looking into it. If your workaround works let us know, that might help narrow things down a little bit.

**Comment by coderbot16** [comment]
This could be an issue with Fabric's registry sync system not properly migrating the IDs between versions.

**Comment by dranorter** [comment]
I can't seem to properly build the 1.15 branch of Traverse. I was able to do it with Terrestria; download, run "gradlew genSources idea", then open the project. But with Traverse I get "Error: Could not find or load main class net.fabricmc.devlaunchinjector.Main". Is there something different? Or maybe I've omitted a step which before I'd done without thinking?

**Comment by dranorter** [comment]
OK, I compiled the 1.15 mod now (my alterations were to comment out lines 29 and 30, ie TraverseBiomes.register() and TraverseGeneration.register()), and two things went differently than I expected. First, once transferred to 1.16 the Traverse biomes did not switch back to being Traverse biomes; for example the autumnal forest was still listed as a birch forest when I press F3.

Secondly, only areas that I visited while using my altered version of the mod were spared the overworld nether nonsense. An autumnal forest I'd generated in under ordinary Traverse, but not visited with my biome-less version, became a Crimson Forest and hoglins started spawning in large numbers.
